src/styles/main.scss

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+}

```

Comment:
It's better not to set margins and paddings using * selector, it's a bit harmful to performance.
Same with list-style-type, are you sure that all the elements have this property?